src/main/java/mod/gate/Gate.java

```diff
@@ -12,7 +13,7 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 
-import static mod.gate.utils.FileUtils.writeJson;
+import mod.gate.utils.FileUtils.*;
```

Comment:
Don't do wildcard imports, stuff likes to break with these, just import FileUtils since you aren't importing the methods statically

Reply:
if you look a little higher it already does it, I just forgot to remove that one

Reply:
the non-used one will be removed in next commit